Automattic/simplenote-android
# Update Checkbox Metrics

### Fix
Update the color and size of checkboxes in the editor to indicate they are tappable and easier to tap, respectively.  All checkboxes in lists including the **_Select Note_** dialog of the **_Note (Light)_** and **_Note (Dark)_** widgets remain unchanged in both color and size.  See the screenshots below for illustration.

![update_checkbox_metrics](https://user-images.githubusercontent.com/3827611/98739432-92b67700-2366-11eb-8ded-047909d8c048.png)

The translucent square behind the checkbox shows the tappable area.  Notice the small space between lines in both **_Before_** and **_After_** columns.  That is the line height spacing value, which applies to all lines in the editor and not only those with checkboxes.

### Test
1. Notice notes with checkboxes remain unchanged in note list.
2. Tap any note with checkboxes in list.
3. Notice checkboxes match accent color of theme (i.e. `simplenote_blue_50`/`#3361cc` for **_Light_** and `simplenote_blue_20`/`#84a4f0` for **_Dark_**).
4. Tap any checkbox in note.
5. Notice checkbox toggles from checked to unchecked or unchecked to checked.
6. Notice area approximately half size of checkbox around checkbox is tappable.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
